### ISSUE TITLE
[Bug] Vertical scrolling in textarea

### ISSUE THREAD

**Issue by NothingWeAre** [quasarframework/quasar#1048]
Can we have vertical scrolling in textarea when content is greater than max-height of said textarea ?
Right now it is not obvious whether textarea currently displaying all text or if some part do not fit.

**Comment by NothingWeAre** [comment]
Rather this should be bug instead of request.
textarea styles
```stylus
textarea {
    overflow: auto;
    overflow-x: auto;
    overflow-y: auto;
}
```
overridden by: 
```stylus
.q-input-target, .q-input-shadow {
    ...........
    overflow: hidden;
    overflow-x: hidden;
    overflow-y: hidden;
    ...........
}
```
as this is more specific selector.

Possible fix:
```stylus
textarea, textarea.q-input-target, textarea.q-input-shadow  {
    overflow: auto;
    overflow-x: auto;
    overflow-y: auto;
}
```

**Comment by rstoenescu** [comment]
Available in v0.15.6 when using the `max-height` prop.

**Comment by GastroGeek** [comment]
@rstoenescu 

Hi,

Just a note - I updated the cli tool and built a brand new project specifically to test the textarea/q-input in .15.6 and still no scrolling? css still shows as 'hidden' too...

Thanks

**Comment by rstoenescu** [comment]
Small mistake in the CLI. It still bundled Quasar 0.15.5. Upgrade to CLI v0.15.6. Sorry for this.

**Comment by GastroGeek** [comment]
Not to be a pain... but still no go?

Global packages
  NPM                   	5.5.1
  yarn                  	1.3.2
  quasar-cli            	0.15.7
  vue-cli               	2.9.3
  cordova               	8.0.0

Important local packages
  quasar-cli            	0.15.7	(Quasar Framework CLI)
  quasar-framework      	0.15.6	(Build responsive websites, PWAs, hybrid mobile apps and Electron

**Comment by rstoenescu** [comment]
I'm not sure how this was missed, but you are right. Fixed it. Will be available in Quasar v0.15.7 (CLI will probably be at 0.15.8).

**Comment by karabo25** [comment]
Hey @rstoenescu 

The text area does not seem scroll on mobile. When you try to scroll on the text area it move the entire web page.
